packages/kbn-apm-synthtrace/src/lib/apm/aggregators/create_breakdown_metrics_aggregator.ts

```diff
@@ -7,13 +7,14 @@
  */
 import { ApmFields } from '@kbn/apm-synthtrace-client';
 import { identity, negate } from 'lodash';
+import { ScenarioOptions } from '../../../cli/scenario';
 import { createFilterTransform, fork } from '../../utils/stream_utils';
 import { createApmMetricAggregator } from './create_apm_metric_aggregator';
 
 const filter = (event: ApmFields) =>
   event['processor.event'] === 'metric' && event['metricset.name'] === 'span_breakdown';
 
-export function createBreakdownMetricsAggregator(flushInterval: string) {
+export function createBreakdownMetricsAggregator(flushInterval: string, options?: ScenarioOptions) {
```

Comment:
I'd prefer the aggregators to not know about ScenarioOptions, e.g. the `createBreakdownMetricsAggregator` takes another option similar to `flushInterval` that is specific to this aggregator. We can change the signature to something like:

```ts
export function createBreakdownMetricsAggregator({ flushInterval, maxGroupCount }: { flushInterval: string; maxGroupCount:number  }) {
```